app/script/view_model/AuthViewModel.coffee

```diff
@@ -1319,7 +1330,10 @@ class z.ViewModel.AuthViewModel
 
       # Show history screen if there are already registered clients
       if client_ets?.length > 0
-        @_set_hash z.auth.AuthView.MODE.HISTORY
+        @_has_local_history()
```

Comment:
History screen has to detect whether the devices was previously used. Doing a simple check on the conversation events to show different copy.